packages/contracts/contracts/foundry/Sphinx.sol

```diff
@@ -141,17 +141,24 @@ abstract contract Sphinx {
 
     function sphinxCollectProposal(
         bytes memory _scriptFunctionCalldata,
-        string memory _deploymentInfoPath
-    ) external {
+        string memory _deploymentInfoPath,
+        uint64 _callDepth
+    ) external returns (FoundryDeploymentInfo memory) {
```

Comment:
We return the `FoundryDeploymentInfo` to make it easier to use `sphinxCollectProposal` in tests